cmd/git-chglog/questioner.go

```diff
@@ -83,6 +83,11 @@ func (q *questionerImpl) ask() (*Answer, error) {
 	var previewableTransform = func(ans interface{}) (newAns interface{}) {
 		if s, ok := ans.(string); ok {
 			newAns = q.parsePreviewableList(s)
+		} else if s, ok := ans.(survey.OptionAnswer); ok {
```

Comment:
Thank you for pointing out this bug and putting in a fix for it. I opened a hotfix PR #113 to address this on it's own.